### ISSUE TITLE
Some of pl-PL keyboard buttons not sent properly 

### ISSUE THREAD

**Issue by Gryxx** [rustdesk/rustdesk#1702]
When connecting form Linux (openSUSE Tumbleweed, KDE Plasma X11 if relevant) to Windows 10 (21H2) i noticed some of the letters getting converted, for example ł to l, ś to s and ó to o, as if alt key was not properly send to remote machine. Issue does not occur if connecting to another Linux client (again, openSUSE Tumbleweed, KDE Plasma X11 if relevant).
Version of software: 1.1.9
Installed from .rpm on Linux, and from .exe on Windows.

I would assume the issue is not specific to pl-PL keyboard layout.

**Comment by rustdesk** [comment]
@asur4s confirm if work on our new version please.

**Comment by asur4s** [comment]
In 1.2.0(unreleased), the ł and ś of polish keyboard work fine. Please look forward to out next version.

**Comment by Gryxx** [comment]
Thanks, looking forward to the new version. Awesome tools BTW, keep the good work.

**Comment by rustdesk** [comment]
try out https://github.com/rustdesk/rustdesk/releases/tag/nightly please.

**Comment by Gryxx** [comment]
Updated on Windows, sending combined characters crashes RustDesk Host on Windows. Will need to compile to update on Linux.

**Comment by rustdesk** [comment]
> crashes

@asur4s

**Comment by Gryxx** [comment]
After compiling newer version on Linux it now works. Please give some indication that connecting from older client can crash RustDesk on remote machine.

**Comment by asur4s** [comment]
Do you remember the trigger for the crash? what key was clicked? @Gryxx

**Comment by Gryxx** [comment]
Alt + o. The version on Linux (client) was on release, Windows (host) was on nightly. Git to nightly worked without crash, but now i noticed that using shift or alt with letters can produce double input.

**Comment by rustdesk** [comment]
> but now i noticed that using shift or alt with letters can produce double input.

1.1.9 (client) -> 1.2.0 nightly (host)?

**Comment by Gryxx** [comment]
No, 1.2.0 to 1.2.0
1.1.9 (client) -> 1.2.0 nightly (host) i can't send shift or alt keys

**Comment by Gryxx** [comment]
Another issue: 1.2.0 to 1.2.0, both linux, cant send alt keys, they work as i would use SHIFT instead

**Comment by rustdesk** [comment]
> alt

@asur4s

**Comment by rustdesk** [comment]
> Win -> Win , `Shift + Delete·`  Shift often doesn't take effect
> Shift + letter keys, ok
> Shift + Delete takes effect only once connected, later not work

**Comment by rustdesk** [comment]
> Under Win, Ctrl, Shift, Win keys do not work sometimes
Combination keys work as separate letter keys when pressed

>  Win -> Win ，Shift + Delete 似乎Shift经常没有生效
Shift + 字母键，生效的
Shift + Delete 有时候刚连接，会生效一次

**Comment by Gryxx** [comment]
> > Win -> Win , Shift + Delete· Shift often doesn't take effect
> > Shift + letter keys, ok
> > Shift + Delete takes effect only once connected, later not work
> 
> Fixed in #1857

Can attest, Linux -> Win now works as expected. Tested left alt and control shortcuts and right alt,  r_alt+shift and shift itself.
Will test Linux->Linux later.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/1547

**Comment by Gryxx** [comment]
> Will test Linux->Linux later.

Tested, seems to work fine.

**Comment by asur4s** [comment]
This issue should be closed
